### PR TITLE
feat: update to `@ui5/webcomponents@1.21.0`

### PR DESCRIPTION
**Pull request by Lukas742**

### CODE REVIEW

File: packages/main/src/webComponents/Select/index.tsx [hunk 1]
```diff
@@ -57,11 +56,12 @@ export interface SelectDomRef extends Omit<SelectAttributes, 'menu'>, Ui5DomRef
    */
   value?: string;
   /**
-   * Defines a reference (ID or DOM element) of the component's options menu, as an alternative to defining the selection's drop-down menu.
-   *
-   * **Note:** Usage of `SelectMenu` is recommended.
+   * Defines the value state of the component.
    */
-  menu?: string | HTMLElement;
```
Comment (MarcusNotheis) [inline packages/main/src/webComponents/Select/index.tsx:64]: I think the menu getter from the dom ref can also be a HTML Element, can't it?

File: packages/main/src/webComponents/ToolbarSeparatorV2/index.tsx [hunk 2]
```diff
@@ -48,7 +44,7 @@ const ToolbarSeparatorV2 = withWebComponent<ToolbarSeparatorV2PropTypes, Toolbar
 
 ToolbarSeparatorV2.displayName = 'ToolbarSeparatorV2';
 
-ToolbarSelect.defaultProps = {
+ToolbarSeparatorV2.defaultProps = {
```
Comment (MarcusNotheis) [inline packages/main/src/webComponents/ToolbarSeparatorV2/index.tsx:47]: 🙈

**Review by MarcusNotheis — APPROVED** [review]

File: packages/main/src/webComponents/Select/index.tsx (hunk 1 above)
Comment (Lukas742) [reply packages/main/src/webComponents/Select/index.tsx]: Good catch!
I also forgot to move the `value` getter/setter to the DomRef again.

**Review by MarcusNotheis — APPROVED** [review]
🚢

**Pull request merged**